Kernel/FileSystem/VirtualFileSystem.h

```diff
@@ -120,7 +125,7 @@ class VirtualFileSystem {
     SpinlockProtected<RefPtr<Custody>, LockRank::None> m_root_custody {};
 
     SpinlockProtected<IntrusiveList<&Mount::m_vfs_list_node>, LockRank::None> m_mounts {};
-    SpinlockProtected<IntrusiveList<&FileBackedFileSystem::m_file_backed_file_system_node>, LockRank::None> m_file_backed_file_systems_list {};
+    MutexProtected<IntrusiveList<&FileBackedFileSystem::m_file_backed_file_system_node>> m_file_backed_file_systems_list {};
```

Comment:
Why do we go from a spinlock to a mutex? I don't understand the relation to the rest of this commit.

Reply:
I will explain this with a comment and in the commit message.